TelegramBotAPI/Controllers/OfficeController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace PresentationLayer.Controllers
+{
+    [Route("api/[controller]")]
+    
+    public class OfficeController : ControllerBase
```

Comment:
Add ApiController attribute and instead of Controller base , use Controller as a parent

Reply:
Done